app/models/member.rb

```diff
@@ -138,7 +138,7 @@ def male?
   end
 
   def formatted_birthday
-    birthday.strftime("%m/%d/%Y")
+    birthday&.strftime("%m/%d/%Y")
```

Comment:
I'd argue you shouldn't even be calling `formatted_birthday` if `birthday` is nil. Also, if we're going to add additional logic to this method, I think we should unit test it.

Reply:
Since we're testing this at the `medicaid_application_member_attributes_spec` level, would it make more sense to have this as:
```
if member.birthday.present?
  member_attributes[:"#{position}_member_birthday"] => member.formatted_birthday,
end
```

up higher?